com.xargon.retrodeck.yml

```diff
@@ -0,0 +1,643 @@
+app-id: com.xargon.retrodeck
```

Comment:
Still the question remain. `xargon.com` is "for sale" that mean it's not under the control of the developers.

Reply:
Now I own retrodeck.net, can it be `net.retrodeck.retrodeck` or `net.xargon.retrodeck`?

Reply:
It's all explained here
https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
now is `net.retrodeck.retrodeck` that I own: https://retrodeck.net